### ISSUE TITLE
Why you pretrained VGG19-54?

### ISSUE THREAD

**Issue by SlimRG** [xinntao/BasicSR#200]
1. What VGG19 doing with image? 
2. Should I train my own VGG19 or better use pretrained?
3. Can I replace VGG19 with anouther network model?

**Comment by SlimRG** [comment]
4. Is VGG19 changhed while train.py is runnig?
P.S> Sorry for my English, I speak by heart :3

**Comment by SlimRG** [comment]
5. Where in ESRGAN do you use VGG19

**Comment by xinntao** [comment]
1. VGG19 is used as the perceptual loss in SRGAN/ESRGAN.
2. The pre-trained model is enough.
3. You can have a try, such as ResNet or others. I did not find large improvements when using ResNet.
4. No
5. The perceptual loss, netF.

**Comment by SlimRG** [comment]
Thanks
